### PR TITLE
🚚 Rename Combobox to Autocomplete

### LINKED ISSUE #1911: Rename Combobox to Autocomplete

### PR DESCRIPTION
**Pull request by mimarz**
resolves #1911

### CODE REVIEW

File: packages/eds-lab-react/src/components/Autocomplete/Autocomplete.tsx [hunk 1]
```diff
@@ -265,7 +270,7 @@ function ComboboxInner<T>(
     setSelectedItems,
   } = useMultipleSelection(multipleSelectionProps)
 
-  let comboBoxProps: UseComboboxProps<ComboboxOption<T>> = {
+  let comboBoxProps: UseComboboxProps<AutocompleteOption<T>> = {
```
Comment (oddvernes) [inline packages/eds-lab-react/src/components/Autocomplete/Autocomplete.tsx:273]: Forgot this?

Comment (mimarz) [reply packages/eds-lab-react/src/components/Autocomplete/Autocomplete.tsx:273]: These are the props/options that goes to downshifts `useCombobox` hook.

Comment (oddvernes) [reply packages/eds-lab-react/src/components/Autocomplete/Autocomplete.tsx:273]: Yea I just realized looking a bit more closely

**Review by oddvernes — APPROVED** [review]
LGTM! 👍

**Pull request merged**